src/backends/pytorch/libtorch_backend.cc

```diff
@@ -175,6 +175,10 @@ LibTorchBackend::CreateExecutionContexts(
         Run(runner_idx, payloads, func);
       }));
 
+  if (Config().has_model_warm_up()) {
+    RETURN_IF_ERROR(WarmUp());
```

Comment:
Based on the thread based caching we see here (https://github.com/pytorch/pytorch/issues/27902), it is possible that this will not be reliable. We should document or call this out somewhere.